### ISSUE TITLE
'fsserver stop' generates apparently extraneous output

### ISSUE THREAD

**Issue by wehimwich** [nvi-inc/fs#80]
For example:

```
{
 "jsonrpc": "2.0",
 "id": "cli-1476363156",
 "result": {}
}
```

**Comment by dehorsley** [comment]
That is a "feature" 😄 

All the fsserver cli commands talk to the running server and receive a json response. The "stop/shutdown" procedure doesn't return any results — hence the result field is empty. In principle could error, but it hasn't done so here either so it's all good. If it had, there would be an error field describing what's happened.

Mostly the cli was designed for programmer use, but if we wanted to make it a user facing tool we could unpack the json into a meaningful sentence.

Note the same applies for all fsserver commands (`fsserver window open`, etc).

**Comment by wehimwich** [comment]
Fair enough. The only reason this came up was that to change the value of an environment variable that the FS sees, it is necessary to restart the server. That is sort of a user land function. This appears to be a consequence of the server now continuing to run after a FS termination.  A way to finesse this might be to add a command line option when starting the FS to force a restart of the server if it is already running. That might be also be useful as a way to make sure there is a "complete" restart of the FS.
